.github/workflows/close-pending.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@master
       - name: Execute close-pending script
-        if: github.repository == 'apache/hadoop-ozone'
+        if: github.repository == 'apache/ozone'
         run: ./.github/close-pending.sh
```

Comment:
`close-pending.sh` to has mentions of `hadoop-ozone` should we change there as well.
Like
```
     "https://api.github.com/search/issues?q=repo:apache/hadoop-ozone+type:pr+updated:<$(date -d "-21 days" +%Y-%m-%d)+label:pending+is:open" \
```

Reply:
Nice catch, thanks.  I think it might work due to the redirect, but it's better to change it anyway.